---a/net/minecraft/util/IObjectIntIterable.java
+++b/net/minecraft/util/IObjectIntIterable.java
@@ -1,5 +1,5 @@
 package net.minecraft.util;
 
-public interface IObjectIntIterable extends Iterable
-{
-}
+public interface IObjectIntIterable extends Iterable {
+
+}
